content/en/docs/demo/docker-deployment.md

```diff
@@ -2,15 +2,16 @@
 title: Docker deployment
 linkTitle: Docker
 aliases: [/docs/demo/docker_deployment]
-cSpell:ignore: otelcollector otlphttp
+cSpell:ignore: otelcollector otlphttp spanmetrics
 ---
 
 ## Prerequisites
 
 - Docker
 - [Docker Compose](https://docs.docker.com/compose/install/#install-compose)
   v2.0.0+
-- 4 GB of RAM for the application
+- Make (optional)
```

Comment:
Make isn't optional for the updated instructions, so drop that qualifier:

```suggestion
- Make
```

Reply:
it actually is, if you do not have `make` you can use:
```
docker compose up --force-recreate --remove-orphans --detach
```

Reply:
Resolving this, since we do provide instructions on how to run without Make by using `docker compose ...`